torch/export/_trace.py

```diff
@@ -1612,7 +1612,7 @@ def set_missing_meta_vals(gm, flat_args, num_params_buffers, constant_attrs):
             index += 1
         if node.op == "get_attr" and "val" not in node.meta:
             val = _get_attr(gm, node.target)
-            if val in constant_attrs and isinstance(val, torch.Tensor):
+            if isinstance(val, torch.Tensor):
```

Comment:
Can you update the docstring of this helper function because now we don't care if it is a constant or not anymore? Maybe we can check if the val is not in param/buffers at least?

Reply:
Sure, I can update the docstring. This path is only taken when a meta val doesn't exist, so it boils down to asserting that we've already handled params/buffers.